UnityProject/Assets/Scripts/Electricity/Engine/PowerControlConsole.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PowerControlConsole : MonoBehaviour, IAPCPowered
+{
+	public ResistanceSourceModule ThisDevice;
+	public ReactorControlConsole ControlConsole;
+	private float TargetsSupplyOutputVoltage = 275000;
+
+	private float PreviousVoltage = 0;
+
+	void IAPCPowered.PowerNetworkUpdate(float Voltage)
```

Comment:
needed?

Reply:
```
{
		/*ElectricityFunctions.WorkOutActualNumbers(ThisDevice.ControllingNode.Node.InData);
		var ControllerMultiplier = (TargetsSupplyOutputVoltage+1) / (ThisDevice.ControllingNode.Node.InData.Data.ActualVoltage+1);
		Logger.Log("in ControllerMultiplier " + ControllerMultiplier+"  >> " +  (TargetsSupplyOutputVoltage+1)+ " / " + (ThisDevice.ControllingNode.Node.InData.Data.ActualVoltage+1));
		if (ControllerMultiplier != 0 && !float.IsInfinity(ControllerMultiplier))
		{
			ControlConsole.RequestRelativeChange(ControllerMultiplier);
		}
		PreviousVoltage = ThisDevice.ControllingNode.Node.InData.Data.ActualVoltage;*/

	}
```

Reply:
yeah, It's not used for anything currently was meant for automatically increasing the output of the reactor but is a bit more tricky to calculate then I realised Should I remove it?

Reply:
Guess it depends on whether you want to come back and try again, no harm keeping it in if thats the case, though maybe add a comment on it just so you dont forget why its commented out :p